docs/functions.rst

```diff
@@ -1101,10 +1106,12 @@ Arguments:
    `securityGroupID`, No, `[]string`, list of ``securityGroupID`` to be passed to temporary RDS instance
    `namespace`, No, `string`, namespace in which to execute. Required if ``snapshotID`` is nil
    `dbEngine`, No, `string`, one of the RDS db engines. Supported engines: ``PostgreSQL`` ``aurora`` ``aurora-mysql`` and ``aurora-postgresql``. Required if ``snapshotID`` is nil or Aurora is run in RDS instance
+   `dbSubnetGroup`, No, `string`, DB Subnet Group to be passed to temporary RDS instance
```

Comment:
hmm, I am not sure about it. the description of the function does mention this in brief.
```
This function restores the RDS DB instance either from an RDS snapshot or from the
data dump (if `snapshotID` is not set) that is stored in an object storage.
```

Reply:
since you recently added these args in the function how are you using these fields in your implementation. We might be creating temp RDS instance while restoring database from dump.

Reply:
yeah that's right!!. what is the concern here?

Reply:
Have modified the word `temporary` -> `restored`